docs/en/stack/security/authorization/privileges.asciidoc

```diff
@@ -48,6 +48,9 @@ roles of the user who created or updated them.
 `manage_pipeline`::
 All operations on ingest pipelines.
 
+`manage_saml`::
+TBD
```

Comment:
```suggestion
Enables the use of internal {es} APIs to initiate and manage SAML authentication on behalf of other users.
```

Reply:
Thanks @tvernum I've added that information